__tests__/partials.ts

```diff
@@ -15,7 +15,7 @@ test('it allows for injecting an HTML partial', async () => {
     },
   });
   const result = await build(temp.dir, {
-    partialDirectory: temp.path('partials'),
+    partialDirectory: [temp.path('partials')],
```

Comment:
I don't think I want to make this a breaking change; we'll want to ensure that if a single path is provided, then things works like before, but if an _array_ of paths are provided, all of them are considered locations that partials could live.